lib/http_server/router.ex

```diff
@@ -1,17 +1,19 @@
-defmodule Router do
-    alias Response.Builder
-    alias Request.Parser
+defmodule HttpServer.Router do
+    alias HttpServer.Exception
+    alias HttpServer.Request.Parser
+    alias HttpServer.Response
+    alias HttpServer.Response.Builder
 
-    def handle_request(http_message, routes \\ Routes) do
+    def handle_request(http_message, routes) do
```

Comment:
If we move `Routes` out of `HttpServer.HttpServer` 
```suggestion
    def handle_request(http_message) do
```